sdk/monitor/Azure.Monitor.Query/src/Generated/Models/NamespaceClassification.cs

```diff
@@ -30,8 +31,10 @@ public NamespaceClassification(string value)
         public static NamespaceClassification Platform { get; } = new NamespaceClassification(PlatformValue);
         /// <summary> Custom. </summary>
         public static NamespaceClassification Custom { get; } = new NamespaceClassification(CustomValue);
+
         /// <summary> Qos. </summary>
-        public static NamespaceClassification Qos { get; } = new NamespaceClassification(QosValue);
+        [CodeGenMember("Qos")]
```

Comment:
You would need to define a partial struct outside of generated directory and applying this attribute there. Otherwise running the generator would reset your changes.